events/tutorials/tutorial_import_user_events_big_query.md

```diff
@@ -0,0 +1,162 @@
+# **Import User Events From the Big Query Table Tutorial**
+
+## Let's get started
+
+Retail API offers you a convenient way to import user event data from previously loaded BigQuery table.
+
+Using BigQuery table allows you to import massive amounts of user events data with no limits.
+
+To find more information about different import types, their restrictions, and use cases, check the [Retail API documentation](https://cloud.google.com/retail/docs/import-user-events#considerations)
+
+**Time to complete**: 
+<walkthrough-tutorial-duration duration="3.0"></walkthrough-tutorial-duration>
+
+## Before you begin
+
+To run Python code samples from this tutorial, you need to set up your virtual environment.
+
+To do that, run the following commands in a terminal:
+```bash
+pip install virtualenv
+```
+```bash
+virtualenv <your-env>
+```
+```bash
+source <your-env>/bin/activate
+```
+Next, install Google packages:
+```bash
+pip install google
+```
+```bash
+pip install google-cloud-retail
+```
+
+**Tip**: Click the copy button on the side of the code box to paste the command in the Cloud Shell terminal to
+run it.
+
+## Set the PROJECT_NUMBER environment variable
+
+As you are going to run the code samples in your own Cloud Project, you should specify the **project_id** as an environment variable, it will be used in every request to the Retail API.
+
+You can find the ```project_number``` in the **Home/Dashboard/Project Info card**.
+
+Set the environment variable with a following command:
```

Comment:
let's also put information about project_id that need to be set

Reply:
done